src/components/ProjectCard/index.jsx

```diff
@@ -53,20 +53,24 @@ export default class ProjectCard extends Component {
     try {
       if (project.icon) {
         const mdiName = pascalCase(project.icon);
-        const ProjectIcon = await import(/* webpackChunkName: "icon" */ `mdi-react/${mdiName}Icon.js`);
+        const ProjectIcon = (await import(/* webpackChunkName: "icon" */ `mdi-react/${mdiName}Icon.js`))
+          .default;
 
-        return this.setState({ projectIcon: <ProjectIcon size={50} /> });
+        return this.setState({
+          projectIcon: <ProjectIcon size={50} />,
+        });
       }
 
-      const projectIcon = await import(/* webpackChunkName: "icon" */ `../../images/projectIcons/${
+      const projectIcon = (await import(/* webpackChunkName: "icon" */ `../../images/projectIcons/${
         project.fileName
-      }.svg`);
+      }.svg`)).default;
 
       this.setState({
         projectIcon: <img height="45" src={projectIcon} alt="Project Icon" />,
```

Comment:
Is there a reason why we are using `<img /> ` to render the icon here and in the `catch` statement we are using a React element?

Reply:
IIRC, in this case we are trying to get the project image from `src/images/projectIcons` folder if it exists. Otherwise if it throws error (the file does not exist), we use the WebIcon as the default icon. However, now it seems like none of the ProjectCard uses the default icon